dropbox-sdk-dotnet/Examples/OauthBasic/Program.cs

```diff
@@ -199,20 +220,29 @@ private async Task<string> GetAccessToken()
                     Console.WriteLine("Finished Exchanging Code for Token");
                     // Bring console window to the front.
                     SetForegroundWindow(GetConsoleWindow());
-
                     accessToken = tokenResult.AccessToken;
                     refreshToken = tokenResult.RefreshToken;
                     var uid = tokenResult.Uid;
                     Console.WriteLine("Uid: {0}", uid);
                     Console.WriteLine("AccessToken: {0}", accessToken);
-                    Console.WriteLine("RefreshToken: {0}", refreshToken);
-                    Console.WriteLine("ExpiresAt: {0}", tokenResult.ExpiresAt);
-
-                    //Settings.Default.AccessToken = accessToken;
-                    Settings.Default.RefreshToken = refreshToken;
+                    if (tokenResult.RefreshToken != null)
+                    {
+                        Console.WriteLine("RefreshToken: {0}", refreshToken);
+                        Settings.Default.RefreshToken = refreshToken;
+                    }
+                    if (tokenResult.ExpiresAt != null)
+                    {
+                        Console.WriteLine("ExpiresAt: {0}", tokenResult.ExpiresAt);
+                    }
+                    if (tokenResult.ScopeList != null)
+                    {
+                        Console.WriteLine("Scopes: {0}", String.Join(" ", tokenResult.ScopeList));
+                    }
+                    Settings.Default.AccessToken = accessToken;
                     Settings.Default.Uid = uid;
-
                     Settings.Default.Save();
+                    http.Stop();
+                    return uid;
```

Comment:
If the function returns `uid` then I think the function name and doc should be changed to reflect that.

Reply:
I can update name and doc to reflect this